auslib/web/base.py

```diff
@@ -41,9 +41,8 @@ def generic(error):
     a 200 response with no updates is returned, because that's what the client
     expects. See bugs 885173 and 1069454 for additional background."""
 
-    if not isinstance(error, BadDataError):
-        if sentry.client:
-            sentry.captureException()
+    if isinstance(error, BadDataError):
+        raise BadDataError
```

Comment:
We still want to send BadDataErrors to Sentry, so please don't remove that. I think the only change needed in this function is re-raise the orginal error instead of only doing it sometimes.